### DISCUSSION TITLE
WaW/Bo1 sound pack (PC only)

### DISCUSSION THREAD

**Discussion by ToonLink0001** [nzp-team/nzportable#689] [Other Content Releases]
This pack includes audio from CoD WaW and Bo1, the perk jingles, announcer, round change and game over. If you want your game to more like the og, this is the mod pack for you!

https://drive.google.com/drive/folders/1rD_FFOrCuRL3y0_OOa2Vk8K9YOVhSIWI?usp=drive_link

**Comment by StickyPuddingg** [comment]
Thank you for your contribution, this is just what i was looking for! ❤️

**Comment by Tadeoelbeo** [comment]
Is it possible to make a psp port?

**Reply by ToonLink0001** [reply]
idk how to

**Reply by ItzDDB** [reply]
yo bro do the oldd voice effect on everything beacuse the zombies sound so bad
